cli/api/daemon.go

```diff
@@ -713,15 +713,23 @@ func (d *daemon) startAgent() error {
 func (d *daemon) registerMasterRPC() error {
 	glog.V(0).Infoln("registering Master RPC services")
 
-	if err := d.rpcServer.RegisterName("Master", master.NewServer(d.facade)); err != nil {
+	server := master.NewServer(d.facade)
+	bypass := os.Getenv("BYPASS_LOCAL_RPC")
```

Comment:
So this is an opt-in model?

If it works and we trust, wouldn't we want it used by default, and just leave ourselves an escape hatch if we find an unanticipated problem later?